### PR TITLE
fix: add akshatcoder-hash

### LINKED ISSUE #1391: Akshat Sharma
## Profile Screenshot

image

### PR DESCRIPTION
**Pull request by akshatcoder-hash**
resolves #1391 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/akshatcoder-hash.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (Panquesito7) [inline profiles/akshatcoder-hash.json:1]: There's an extra line here.
```suggestion
```

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

---
**Summary:** :ok_hand: This code looks good

**Comment by schmelto** [comment]
@Panquesito7 can you merge it I'm currently only on my phone

**Review by Pradumnasaraf — COMMENTED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**